crates/next-custom-transforms/tests/fixture/next-dynamic-app-dir/no-ssr/output-turbo-dev.js

```diff
@@ -0,0 +1,17 @@
+import { __turbopack_module_id__ as id } from "../text-dynamic-no-ssr-server" with {
+    "turbopack-transition": "next-client-dynamic",
+    "turbopack-chunking-type": "none"
```

Comment:
Does it work in the single graph when it doesn't have a chunking type?

Reply:
Yes. This subgraph is still visited when the `EcmascriptClientReference` reference with `ChunkingType::Isolated` is walked.